rust/lance-index/src/scalar/bitmap.rs

```diff
@@ -403,3 +469,147 @@ pub async fn train_bitmap_index(
 
     do_train_bitmap_index(batches_source, dictionary, index_store).await
 }
+
+#[cfg(test)]
+pub mod tests {
+    #[tokio::test]
+    #[ignore]
```

Comment:
Confirm it passes on my macos 14.4
lance git:(3515-test-read-logic) ✗ RUST_BACKTRACE=1  cargo test scalar::bitmap::tests::test_big_bitmap_index -- --exact --nocapture
```
test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 48 filtered out; finished in 0.00s

     Running unittests src/lib.rs (target/debug/deps/lance_index-20e80213e6b13454)

running 1 test
test scalar::bitmap::tests::test_big_bitmap_index has been running for over 60 seconds
test scalar::bitmap::tests::test_big_bitmap_index ... ok


```